src/factories/Axis.ts

```diff
@@ -6,11 +6,13 @@ module n3Charts.Factory {
     public svg: D3.Selection;
     public scale: D3.Scale.Scale;
     public axis: D3.Svg.Axis;
+    public static SIDE_X: string = 'x';
```

Comment:
love this. Maybe we can also rewrite the comparisons `=== 'x'` using this static properties